composer.json

```diff
@@ -67,5 +68,10 @@
         "branch-alias": {
             "dev-master": "2.0.x-dev"
         }
+    },
+    "scripts": {
```

Comment:
@soyuka It will not work for my Docker Compose set-up. I need to use copy, not symlink because of all the volumes... And then of course there's Windows. :smile: